configs/katello/4.7.yaml

```diff
@@ -0,0 +1,110 @@
+---
+:project: katello
+:github_org: katello
+:releases:
+  :4.7:
+    :redmine_version_id: 1613
+:prior_releases:
+  :4.6:
+    :redmine_version_id: 1561
```

Comment:
I noticed the version was still open, but since it was released it should be closed. I now did that but perhaps something is missing in the process.